splipy/utils/NACA.py

```diff
@@ -82,11 +82,19 @@ def thickness(x):
         a1 = -0.126
         a2 = -0.3516
         a3 = 0.2843
-        a4 = -0.1015
+        a4 = -0.1036 if closed else -0.1015
```

Comment:
Would you look at that. The default NACA profiles aren't even defined as closed loops :fearful: